pyproject.toml

```diff
@@ -1,7 +1,83 @@
 # Build-system section
 [build-system]
-requires = ["setuptools>=46.4.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling>=0.18.0"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tuf"
+description = "A secure updater framework for Python"
+readme = "README.md"
```

Comment:
does this lead to pypi using the README as project description?

Reply:
yes